### PR TITLE
Configure security scans

### PR DESCRIPTION
**Pull request by VOID404**
**Description**

Changes proposed in this pull request:

- configure trivy security scans

**Related issue(s)**
https://github.com/kyma-project/test-infra/pull/7401

### CODE REVIEW

**Comment by VOID404** [comment]
[Sample logs](https://status.build.kyma-project.io/view/gs/kyma-prow-logs/pr-logs/pull/kyma-project_compass-manager/3/void404_test_of_prowjob_pre-main-compass-manager-trivy/1642854930312073216)

**Comment by VOID404** [comment]
This is basically default trivy config, with added `--no-progress` and `--exit-code 1`

File: .trivy.yaml [hunk 1]
```diff
@@ -0,0 +1,225 @@
+cache:
+    backend: fs
+    clear: false
+    dir: /Users/I354967/Library/Caches/trivy
```
Comment (Disper) [inline .trivy.yaml:4]: I'm not sure whether it's a valid directory as it points to your local dir.

File: .trivy.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+cache:
+    backend: fs
+    ttl: 0s
+db:
+    no-progress: true
+exit-code: 1
+format: table
+ignorefile: .trivyignore
```
Comment (mvshao) [inline .trivy.yaml:8]: Is a `.trivyignore` file necessary if we are not going to ignore selected CVEs?

**Review by mvshao — APPROVED** [review]

**Pull request merged**